src/main/java/net/homework/webapp/service/mapper/DepartmentDtoService.java

```diff
@@ -0,0 +1,25 @@
+package net.homework.webapp.service.mapper;
+
+import net.homework.webapp.dto.DepartmentDto;
+import net.homework.webapp.model.Department;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DepartmentDtoService {
```

Comment:
rename to DepartmentMapper, and make methods static and remove @Service

Reply:
changed